tests/data/preferences/explodeLaunchHandlerStart.json5

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
These 4 json5 files in tests/data/preferences folder can be removed if https://github.com/GPII/universal/pull/615 is merged before this pull request. The same 4 files in testData/preferences should serve the purpose.